Kickstarter-iOS/ViewModels/AppDelegateViewModelTests.swift

```diff
@@ -650,7 +650,7 @@ final class AppDelegateViewModelTests: TestCase {
 
     let params = .defaults
       |> DiscoveryParams.lens.sort .~ .newest
-    self.goToDiscovery.assertValues([params])
+    XCTAssertEqual(self.goToDiscovery.values.first??.queryParams, params.queryParams)
```

Comment:
These tests were only comparing on `queryParams` before so I've kept them that way.